### ISSUE TITLE
Supported Browsers

### ISSUE THREAD

**Issue by allymparker** [red-gate/Tech-Radar#20]
Currently monitor supports:

Chrome
FireFox
Edge
IE 11

via Babel's `Env` preset and `babel-polyfill`

**Comment by MichelleTaylorRG** [comment]
https://info.red-gate.com/eng/development-policies/platforms-and-supported-versions/browsers exists, but doesn't look like it's actually very up to date

**Comment by tugberkugurlu** [comment]
FWIW, we recently dropped support for IE, hasn't heard any complaints from EAP co-creators for Data Catalog. There is an element if surprise but that's easily communicated, there has been no resistance to switching to an alternative evergreen browser such as Chrome, Edge, Firefox, etc. We are at the EAP stage though, so, it may be misleading.

More info here around the rationale: https://github.com/red-gate/Mosaic/blob/master/.adr/2018-09-13-Drop-support-for-IE-11.md

**Comment by ChrisHurley** [comment]
That's the same set we currently support in Clone. IE11 is the only one that we really have to spend any time on specifically.

**Comment by fffej** [comment]
@MichelleTaylorRG That page is up to date - I think we should only support evergreen browsers (e.g. browsers that upgrade themselves).

[IE 11](https://netmarketshare.com/browser-market-share.aspx?options=%7B%22filter%22%3A%7B%22%24and%22%3A%5B%7B%22deviceType%22%3A%7B%22%24in%22%3A%5B%22Desktop%2Flaptop%22%5D%7D%7D%5D%7D%2C%22dateLabel%22%3A%22Trend%22%2C%22attributes%22%3A%22share%22%2C%22group%22%3A%22browserVersion%22%2C%22sort%22%3A%7B%22share%22%3A-1%7D%2C%22id%22%3A%22browsersDesktopVersions%22%2C%22dateInterval%22%3A%22Monthly%22%2C%22dateStart%22%3A%222017-11%22%2C%22dateEnd%22%3A%222018-10%22%2C%22segments%22%3A%22-1000%22%7D) makes up less than 10% of usages and is on a (slow) but steady downward trend.

**Comment by fffej** [comment]
Paging @edwardcasbon for the web team perspective. I was hoping to get stats on what browsers people use to visit www.red-gate.com, but GA is being less than friendly at the mo'.

**Comment by edwardcasbon** [comment]
Thanks @fffej . Since August 1st (so 3 months) the browser split has been as follows:

- Chrome: 70.13%
- IE: 10.25%
	- 11: 94.10%
	- 7.0: 4.91%
	- 9.0: 0.48%
- Firefox: 8.67%
- Edge: 4.84%
- Safari: 3.80%
- Opera: 0.84%

When developing for the Redgate website we aim to support evergreen browsers (latest versions of Chrome, Firefox and Edge) + IE 11.

**Comment by ChrisHurley** [comment]
The reason we've been concerned with IE11 support has primarily been because it's the standard browser on an OS we still support (Windows 7/Server 2008 R2) and we weren't sure if people have Chrome/Firefox installed on servers, if they wanted to access Clone from them.

**Comment by fffej** [comment]
Let's favor consistency across Redgate. I think that means:

Adopt:
* Chrome
* FireFox
* Edge
* IE 11

Retire:
- IE 10 and less

Any disagreements?

**Comment by tugberkugurlu** [comment]
> Adopt:
> 
> * IE 11

Does this mean we have to support all these, including IE 11? TBH, we don't have any intention to support IE 11 for Data Catalog as we don't have anyone blocked on this.

**Comment by fffej** [comment]
@tugberkugurlu There's a strong benefit for us to have a single consistent story about where our software works and where it doesn't. ~10% of customers still use IE 11 which seems significant enough not to ignore.

In terms of what to be about new products such as Data Catalog - I think it's fine to descope support for IE11 now (since finding product/market fit is more important!), but before we release v1 we should do the work to support IE11.

**Comment by tugberkugurlu** [comment]
Thanks @fffej, sure, I understand the rationale and I am happy with it. However, the consequences are:

 - We will spend extra time trying to make this work. Considering our initial try, this wasn't straight forward with well-known approached such as pollyfilling (more info [here](https://github.com/red-gate/Mosaic/issues/165#issuecomment-421064636)).
 - It will gradually get worse as the time progress due to IE 11 being superseded by Edge.

> ~10% of customers still use IE 11

I don't know where we get this number but it was around ~5% for SQL Clone when we looked, see: https://github.com/red-gate/Mosaic/blob/master/.adr/2018-09-13-Drop-support-for-IE-11.md

As long as we understand these consequences, it's OK for us to take the cost and make it work for IE11 even if there is not going to be real benefit to us as a business (due to customer being happy to go with an alternative browser).

**Comment by tugberkugurlu** [comment]
> ~I don't know where we got this number but it was around 5% for SQL Clone when we looked, see: https://github.com/red-gate/Mosaic/blob/master/.adr/2018-09-13-Drop-support-for-IE-11.md

sorry, found it now https://github.com/red-gate/Tech-Radar/issues/20#issuecomment-435339684

**Comment by fffej** [comment]
We'll revisit this issue when we next review the Tech Radar, but for now we've gone for broad compatability across all products and included IE11 in the support.

Thanks to all for the great discussion around this!
